lib/src/device_connector.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_reactive_ble/src/generated/bledata.pb.dart' as pb;
+import 'package:flutter_reactive_ble/src/model/connection_state_update.dart';
+import 'package:flutter_reactive_ble/src/rx_ext/repeater.dart';
+import 'package:meta/meta.dart';
+
+import 'model/uuid.dart';
+
+class DeviceConnector {
+  const DeviceConnector({
+    @required MethodChannel bleMethodChannel,
```

Comment:
I would definitely prefer to manage all the IPC in a single place (like in the facade, even though it's questionable, whether a thing called a "facade" is the right place to take care of IPC).

Do you think it's feasible?